### DISCUSSION TITLE
Integration keeps resetting hostname/ip

### DISCUSSION THREAD

**Discussion by ciB89** [SukramJ/aiohomematic#1018] [Q&A]
Whenever I restart HA (e.g. after an update), the ip/hostname of the integration for my CCU instance changes to what I believe is the container ip adresse. The integration then stops working. Only after I manually reset it to the right ip adresse it starts working again. This a new behaviour, and I don't know why this is happening. Any advise?

**Comment by danielperna84** [comment]
You should use the __hostname__ of your CCU instance. That way it doesn't matter if the IP changes.

**Reply by ciB89** [reply]
Well, the ip (which is my static ip of my raspberrypi) does not change, the integration changes the ip I entered by itself without me changing anything just by restarting home assistant.
